tools/restatectl/src/commands/log/mod.rs

```diff
@@ -19,4 +20,6 @@ pub enum Log {
     List(list_logs::ListLogsOpts),
     /// Get the details of a specific log
     Describe(describe_log::DescribeLogIdOpts),
+    /// Dump the contents of a bifrost log
+    Dump(dump_log::DumpLogOpts),
```

Comment:
How would you feel about introducing a new top-level `dump` subcommand, of which this is one flavor? Then we can group all the tools that deal with DB directly without requiring a running Restate cluster can get grouped under `restatectl dump`.